src/controllers/clients/delete.action.ts

```diff
@@ -43,3 +43,33 @@ export const deleteClient = async (req: HttpRequest, res: Response, next: NextFu
         next(new HttpError(502, 'Could verify removal status for this client.', e));
     }
 };
+
+
+/**
+ * @swagger
+ * /clients/:rat:
+ *   delete:
+ *     tags:
+ *       - Clients
+ *     description: Revokes a client
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: rat
+ *         in: path
+ *         required: true
+ *         type: string
+ *     responses:
+ *       '200':
+ *         description: OK
+ *       '400':
```

Comment:
@JustinaMary 401, 403, 500, 502 are common response codes in my opinion and we should move then to a higher level, so we dont repeat the same thing for every controller.